src/scripts/Events/EventsList.js

```diff
@@ -0,0 +1,27 @@
+import { getEvents, useEvents } from "./EventDataProvider.js"
+import {eventsHTML} from "./Events.js"
+
+
+
+
+const contentTarget = document.querySelector(".contentContainer")
+//const eventHub = document.querySelector(".container")
+
+
+const render = () => {
+    const events = useEvents()
+
+    const rep = events.map(event => {
+        return eventsHTML(event)
+    })
+    contentTarget.innerHTML = rep
```

Comment:
You never did a .join("") on the map so when this is rendered to innerHTML you get the commas showing up in the DOM